Description: Add dev-dependencies
 Something went wrong with canonicalization of Cargo.toml and the
 dev-dependencies got lost
Author: NoisyCoil <noisycoil@debian.org>
Forwarded: not-needed
Last-Update: 2025-09-28
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -64,3 +64,5 @@
 version = "0.13.0"
 
 [dev-dependencies]
+yoke = "0.8"
+zerovec = "0.11"
